tests/test_upload.py

```diff
@@ -12,6 +14,13 @@
 access_token = 'pk.{0}.test'.format(
     base64.b64encode(b'{"u":"testuser"}').decode('utf-8'))
 
+UUID1 = 4  # chosen by fair dice roll, guaranteed to be random
```

Comment:
But then I can't make my [xkcd joke](https://imgs.xkcd.com/comics/random_number.png) :-P 

seriously though, it's probably good idea to make the mock uuid more realistic 👍